### ISSUE TITLE
Error during prediction on channel=3 RGB tiff images

### ISSUE THREAD

**Issue by saanasum** [mpicbg-csbd/stardist#6]
After successful addressing #5 a potential follow-up error occurred. When calling `prob, dist = model_no_sc.predict(img)` following error message is displayed:

```python3
UnboundLocalError                         Traceback (most recent call last)
<timed exec> in <module>

/usr/local/lib/python3.5/dist-packages/stardist/model.py in predict(self, img, resizer, **predict_kwargs)
    454         if img.ndim == 2:
    455             x = np.expand_dims(img,channel)
--> 456         self.config.n_channel_in == x.shape[channel] or _raise(ValueError())
    457 
    458         # resize: make divisible by power of 2 to allow downsampling steps in unet

UnboundLocalError: local variable 'x' referenced before assignment
```

**Comment by uschmidt83** [comment]
Should be fixed with commit 97fa94d. Please re-install the Python package as described in #5.

**Comment by saanasum** [comment]
Thank you!
